external/wakaama/examples/client/lwm2mclient.c

```diff
@@ -1676,148 +1016,554 @@ static void process_udpconn(int sockfd, fd_set *readfds, client_data_t data)
 	}
 }
 
-
-lwm2m_context_t *wrapper_get_context(void)
+static void prv_close_sock(void)
 {
-	return lwm2mH;
+	connection_t *connPtr = NULL;
+
+	connPtr = ((client_data_t *)(lwm2mH->userData))->connList;
+
+	while (connPtr != NULL) {
+		close(connPtr->sock);
+		connPtr = connPtr->next;
+	}
 }
 
-int client_object_dump(char *buffer)
+pthread_addr_t client_main(void)
 {
-	if (lwm2mH != NULL) {
-		prv_object_dump(buffer, lwm2mH);
-		return 0;
+	int result;
+	int i;
+
+	const char *localPort = "56830";
+#if 0
```

Comment:
Alright, I added the comment only to remove uncompiled code. If you feel it might be used, then it is best to leave it as is. Thanks for the explanation.